src/1d/Makefile.classic_1d

```diff
@@ -0,0 +1,19 @@
+#get the directory of this makefile
+LIB:=$(dir $(lastword $(MAKEFILE_LIST)))
```

Comment:
I suggest we hardwire this as `LIB = $(CLAW)/classic/src/1d` rather than determining it this way.  In classic this works fine as is, but in geoclaw we need to use `GEOLIB`, `AMRLIB`, and also will probably want to refer to `$(CLAW)/riemann` for the Riemann solvers in the `Makefile.geoclaw_2d_shallow`, for example.